src/core/lib/gprpp/atomic.h

```diff
@@ -21,10 +21,63 @@
 
 #include <grpc/support/port_platform.h>
 
-#ifdef GPR_HAS_CXX11_ATOMIC
-#include "src/core/lib/gprpp/atomic_with_std.h"
-#else
-#include "src/core/lib/gprpp/atomic_with_atm.h"
-#endif
+#include <atomic>
+
+namespace grpc_core {
+
+template <typename T>
+using atomic = std::atomic<T>;
```

Comment:
Maybe capitalize this to be in line with C++ style?

Reply:
Good idea. Done.